app/javascript/packages/verify-flow/services/api.ts

```diff
@@ -1,3 +1,7 @@
+export interface ErrorResponse<Field extends string> {
+  error: Record<Field, [string, ...string[]]>;
```

Comment:
is this this `[string, ...string[]]` type a way to say "`string[]` with length at least 1"?